src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/PersonalisationProvider.java

```diff
@@ -23,19 +23,24 @@
 @Service
 public class PersonalisationProvider {
 
-    private static final String HEARING_CENTRE_ADDRESS = "hearingCentreAddress";
+    private static final String HEARING_CENTRE_ADDRESS_CONST = "hearingCentreAddress";
+    private static final String APPEAL_REFERENCE_NUMBER_CONST = "appealReferenceNumber";
+    private static final String ARIA_LISTING_REFERENCE_CONST = "ariaListingReference";
+    private static final String APPELLANT_GIVEN_NAMES_CONST = "appellantGivenNames";
+    private static final String APPELLANT_FAMILY_NAME_CONST = "appellantFamilyName";
```

Comment:
I think _CONST suffix is redundant and as per the standard naming convention for instance, HEARING_CENTRE_ADDRESS denotes it as a constant.

Reply:
Hi @jkumar-moj thanks for the comment. I suffixed with _CONST because there is already the AsylumCaseDefinition fields used in this class (e.g. APPEAL_REFERENCE_NUMBER, APPELLANT_GIVEN_NAMES, etc.) I wanted to differentiate between the class contants and AsylumCaseDefinition fields. What are your thoughts on how best to differentiate between the two as at the same time eliminate code smells?